src/wallet/rpcwallet.cpp

```diff
@@ -755,6 +755,8 @@ UniValue getbalance(const JSONRPCRequest& request)
             "                     avoid passing this argument.\n"
             "2. minconf           (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
             "3. include_watchonly (bool, optional, default=false) Also include balance in watch-only addresses (see 'importaddress')\n"
+            "4. include_unspendable (bool, optional, default=false) Also include balance for UTXOs which are currently not (but\n"
```

Comment:
Actually, `include_watchonly` is slightly longer, but I think realigning is a good idea yeah.